__tests__/app.test.js

```diff
@@ -86,4 +86,38 @@ describe("GET /api/articles", () => {
             })
         })
     })
+})
+describe("GET /api/articles/:article_id/comments", () => {
+    test('returns an arr of articles', () => {
+        return request(app).get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            const comments = body.comments
+            expect(comments).toBeSorted('created_at', {ascending: true})
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: 1
+                })
+            })
+        })
+    })
+    test('ERR - should return 400 when id is not valid', () => {
```

Comment:
Watch your test description, is it a 400? Don't copy and paste test descriptions and then not update them.